hub2vu/yanolja.py

```
@@ -0,0 +1,1000 @@
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.13.7"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2
+}
```

Ruff (0.13.1)

Found useless expression. Either assign it to a variable or remove it.

(B018)